pontoon/gpt/openai_connection.py

```diff
@@ -0,0 +1,51 @@
+from openai import OpenAI
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(
+        self, english_text, translated_text, characteristic, target_locale_name
+    ):
+        system_messages = {
+            "informal": f"""You will be provided with text in English, along with its machine-generated translation in {target_locale_name}. Your objective is to revise the translation to ensure it utilizes simpler language. Please adhere to the following guidelines to achieve this:
+                    - Clarity is Key: Ensure the translation conveys the original message in the clearest possible manner, without ambiguity or unnecessary complexity.
+                    - Consistent Simplicity: Maintain a consistent level of simplicity throughout the translation.
+                    The goal is to produce a translation that accurately reflects the original English text, but in a way that is more approachable and easier to understand for all {target_locale_name} speakers, regardless of their language proficiency.""",
+            "formal": f"""You will be presented with text in English, accompanied by its machine-generated translation in {target_locale_name}. Your task is to refine the {target_locale_name} translation to ensure it adheres to a higher level of formality. In doing so, consider the following guidelines:
```

Comment:
Why is this part of the message rephrased from the one used for `informal` in line 12? Can't we use the same message and only modifiy the vital part - "utilizes simpler language" vs "adheres to a higher level of formality".

Reply:
Are you suggesting to remove the specific guidelines in the formal prompt? Like the "Adjust the tone" and "Formal addressing"?

Reply:
No, I'm only referring to the differences between lines 12 and 16:
- provided with vs presented with
- along with vs accompanied by
- Your objective vs Your task
- to revise the translation vs to refine the {target_locale_name} translation
- Please adhere to the following guidelines to achieve this vs In doing so, consider the following guidelines